lib/plugins/chatops.rb

```diff
@@ -15,12 +15,14 @@ def notify(message)
       case @service
       when 'slack'
         LOGGER.info("Sending Slack webhook message to #{@url}")
-        Chatops::Slack.new(@channel,
-                           @url,
-                           @user,
-                           message,
-                           http_options: @args[:http_options] || {},
-                           icon_emoji: @args[:icon_emoji]).notify
+        Chatops::Slack.new(
```

Comment:
You might consider moving the `require 'plugins/chatops/slack'` in here. That will only load the integration you want to use. Not a big deal now, but as you start adding more integrations, you might not want them all loaded all the time.